lib/content_scanner.py

```
@@ -0,0 +1,223 @@
+    "У": "Y", "у": "y",
+    # Greek
+    "Α": "A", "α": "a",
+    "Ε": "E", "ε": "e",
+    "Ο": "O", "ο": "o",
+    "Ρ": "P", "ρ": "p",
+}
+
+_CONFUSABLES_RE = re.compile("[" + "".join(_CONFUSABLES.keys()) + "]")
+
+# Zero-width characters stripped to empty (F-08 fix vs TS version that maps to space)
+_ZERO_WIDTH_RE = re.compile(r"[​-‍﻿⁠]")
```

Ruff (0.15.10)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

(PLE2515)